src/libstd/collections/hash/map.rs

```diff
@@ -4194,51 +3379,20 @@ mod test_map {
         assert_eq!(map[&6], 60);
     }
 
-    #[test]
-    fn test_adaptive() {
-        const TEST_LEN: usize = 5000;
-        // by cloning we get maps with the same hasher seed
-        let mut first = HashMap::new();
-        let mut second = first.clone();
-        first.extend((0..TEST_LEN).map(|i| (i, i)));
-        second.extend((TEST_LEN..TEST_LEN * 2).map(|i| (i, i)));
-
-        for (&k, &v) in &second {
-            let prev_cap = first.capacity();
-            let expect_grow = first.len() == prev_cap;
-            first.insert(k, v);
-            if !expect_grow && first.capacity() != prev_cap {
-                return;
-            }
-        }
-        panic!("Adaptive early resize failed");
-    }
-
```

Comment:
Can you give some insight on why you don't think this test should apply anymore?

Reply:
Because hashbrown doesn't do early resize when probe chains get too long. See [this comment](https://github.com/rust-lang/rust/pull/56241#issuecomment-444231188).

Reply:
:+1: